tasks/docker_image_prune.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: 2025 Stefan Haun <tux@netz39.de>
+# SPDX-License-Identifier: MIT
+---
+- block:
+    - name: Copy docker-image-prune.service file
+      ansible.builtin.copy:
+        src: files/docker-image-prune.service
+        dest: /etc/systemd/system/docker-image-prune.service
+        mode: "0644"
+      notify: Reload systemd
+
+    - name: Copy docker-image-prune.timer file
+      ansible.builtin.copy:
+        src: files/docker-image-prune.timer
+        dest: /etc/systemd/system/docker-image-prune.timer
+        mode: "0644"
+      notify: Reload systemd
+
+    - name: Enable and start docker-prune.timer
+      ansible.builtin.systemd:
+        name: docker-image-prune.timer
+        enabled: yes
+        state: started
+
+  when: docker_image_prune
```

Comment:
This can go directly under the "block" line:
```
key-order[task]: You can improve the task key order to: when, block
tasks/docker_image_prune.yml:4 Task/Handler: block/always/rescue 
```